service/app/util/Global.java

```diff
@@ -188,6 +177,9 @@ private void intializeRequestInfo(Context ctx, String userId) {
       messageId = JsonKey.DEFAULT_CONSUMER_ID;
     }
     ctx.flash().put(JsonKey.REQUEST_ID, messageId);
+    if (requestInfo == null) {
```

Comment:
requestInfo is initialised already at line number 46.

Reply:
When u do  mock of static block , It won't initialize any static variable, this changes is done for unit test only , because same variable is directly used inside BaseController , if i won't create instance here , on controller it will throw null pointer exception.

Reply:
Ok